tests/Knp/Menu/Tests/Integration/Symfony/RoutingExtensionTest.php

```diff
@@ -8,16 +8,9 @@
 
 final class RoutingExtensionTest extends TestCase
 {
-    protected function setUp(): void
-    {
-        if (!\interface_exists('Symfony\Component\Routing\Generator\UrlGeneratorInterface')) {
-            $this->markTestSkipped('The Symfony Routing component is not available');
-        }
-    }
-
     public function testCreateItemWithRoute(): void
     {
-        $generator = $this->getMockBuilder('Symfony\Component\Routing\Generator\UrlGeneratorInterface')->getMock();
+        $generator = $this->getMockBuilder(UrlGeneratorInterface::class)->getMock();
```

Comment:
You could simplify code like that even more, if you like:

```suggestion
        $generator = $this->createMock(UrlGeneratorInterface::class);
```

Reply:
I was about to apply such change, but didn't want to overload PR.
Maybe for next one?